lowatt_grdf/main.py

```diff
@@ -69,18 +71,63 @@ def decorator(func: Callback) -> Callback:
                 kwargs["help"] = field.field_info.description
             if field.type_ == bool:
                 kwargs["is_flag"] = True
+            if field.alias.startswith("date_"):
+                if field.alias == "date_consentement_declaree":
+                    # Expect a datetime while it seems more consistent to use a
+                    # simple date.
+                    kwargs["callback"] = _validate_date_as_datetime
+                else:
+                    kwargs["callback"] = _validate_date
+                kwargs["metavar"] = "YYYY-MM-DD"
             click.option(opt, **kwargs)(func)
         return func
 
     return decorator
 
 
-@click.group()
+def _validate_date(ctx: Any, param: Any, value: str) -> str:
```

Comment:
I think this should be written as pydantic validator in models.DeclareAccess no ? So this will also be checked if using the python API.